integrations/slack/client.go

```diff
@@ -34,230 +37,257 @@ var desc = kittehs.Must1(sdktypes.StrictIntegrationFromProto(&sdktypes.Integrati
+		),
+		sdkmodule.ExportFunction(
+			"conversations_history",
+			conversationsAPI.History,
+			sdkmodule.WithFuncDoc("https://api.slack.com/methods/conversations.history"),
+			sdkmodule.WithArgs("channel", "cursor?", "limit?", "include_all_metadata?", "inclusive?", "oldest?", "latest?"),
+		),
+		sdkmodule.ExportFunction(
+			"conversations_info",
+			conversationsAPI.Info,
+			sdkmodule.WithFuncDoc("https://api.slack.com/methods/conversations.info"),
+			sdkmodule.WithArgs("channel", "include_locale?", "include_num_members?"),
+		),
+		sdkmodule.ExportFunction(
+			"conversations_invite",
+			conversationsAPI.Invite,
+			sdkmodule.WithFuncDoc("https://api.slack.com/methods/conversations.invite"),
+			sdkmodule.WithArgs("channel", "users", "force?"),
+		),
+		sdkmodule.ExportFunction(
+			"conversations_list",
+			conversationsAPI.List,
+			sdkmodule.WithFuncDoc("https://api.slack.com/methods/conversations.list"),
+			sdkmodule.WithArgs("cursor?", "limit?", "exclude_archived?", "team_id?", "types?"),
+		),
+		sdkmodule.ExportFunction(
+			"conversations_members",
+			conversationsAPI.Members,
+			sdkmodule.WithFuncDoc("https://api.slack.com/methods/conversations.members"),
+			sdkmodule.WithArgs("channel", "cursor?", "limit?"),
+		),
+		sdkmodule.ExportFunction(
+			"conversations_open",
+			conversationsAPI.Open,
+			sdkmodule.WithFuncDoc("https://api.slack.com/methods/conversations.open"),
+			sdkmodule.WithArgs("channel?", "users?", "prevent_creation?"),
+		),
+		sdkmodule.ExportFunction(
+			"conversations_rename",
+			conversationsAPI.Rename,
+			sdkmodule.WithFuncDoc("https://api.slack.com/methods/conversations.rename"),
+			sdkmodule.WithArgs("channel", "name"),
+		),
+		sdkmodule.ExportFunction(
+			"conversations_replies",
+			conversationsAPI.Replies,
+			sdkmodule.WithFuncDoc("https://api.slack.com/methods/conversations.replies"),
+			sdkmodule.WithArgs("channel", "ts", "cursor?", "limit?", "include_all_metadata?", "inclusive?", "oldest?", "latest?"),
+		),
+		sdkmodule.ExportFunction(
+			"conversations_set_purpose",
+			conversationsAPI.SetPurpose,
+			sdkmodule.WithFuncDoc("https://api.slack.com/methods/conversations.setPurpose"),
+			sdkmodule.WithArgs("channel", "purpose"),
+		),
+		sdkmodule.ExportFunction(
+			"conversations_set_topic",
+			conversationsAPI.SetTopic,
+			sdkmodule.WithFuncDoc("https://api.slack.com/methods/conversations.setTopic"),
+			sdkmodule.WithArgs("channel", "topic"),
+		),
+		sdkmodule.ExportFunction(
+			"conversations_unarchive",
+			conversationsAPI.Unarchive,
+			sdkmodule.WithFuncDoc("https://api.slack.com/methods/conversations.unarchive"),
+			sdkmodule.WithArgs("channel"),
+		),
+
+		// Reactions.
+		sdkmodule.ExportFunction(
+			"reactions_add",
+			reactionsAPI.Add,
+			sdkmodule.WithFuncDoc("https://api.slack.com/methods/reactions.add"),
+			sdkmodule.WithArgs("channel", "name", "timestamp"),
+		),
+
+		// Users.
+		// TODO: sdkmodule.ExportFunction(
```

Comment:
Is there an issue associated with this TODO?

Reply:
Added to the comment: ENG-1057